src/StableMemory.mo

```diff
@@ -0,0 +1,93 @@
+/// Byte-level access to (virtual) _stable memory_.
+///
+/// This is a lightweight abstraction over IC _stable memory_ and supports persisting
+/// raw binary data across Motoko upgrades.
+/// Use of this module is fully compatible with Motoko's use of
+/// _stable variables_, whose persistence mechanism also uses (real) IC stable memory internally, but does not interfere with this API.
+///
+/// Each `load` operation loads from byte address `offset` in little-endian
+/// format using the natural bit-width of the type in question.
+/// The operation traps if attempting to read beyond the current stable memory size.
+///
+/// Each `store` operation stores to byte address `offset` in little-endian format using the natural bit-width of the type in question.
+/// The operation traps if attempting to write beyond the current stable memory size.
+///
+/// Text values can be handled by using `Text.decodeUtf8` and `Text.encodeUtf8`, in conjunction with `loadBlob` and `storeBlob`.
```

Comment:
Is there a good reason not to include these compositions here directly as the implementation of `storeText` and `readText`, rather than omitting those cases? 

(Is there a value in this module being minimal?)

Reply:
I have a similar thought for the other compositions, for type `Principal`.

Perhaps we should provide the "official recipes" in this module?